js-intro.js

```diff
@@ -1,96 +1,175 @@
-// ASSESSMENT 1: INTRO TO JAVASCRIPT
-// Coding practical questions
+// // ASSESSMENT 1: INTRO TO JAVASCRIPT
+// // Coding practical questions
 
-// --------------------------- Consider the following variable:
+// // --------------------------- Consider the following variable:
 
 var mantra = "Be the dev"
-
-// 1a. Write the code that determines if there is a 'B' in mantra.
-
-
-
-// 1b. Write the code that determines if there is an 'x' in mantra.
-
-
-
-// 1c. Write the code that determines if there is a 'v' in mantra.
-
-
-
-// 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
-
-
-
-// ------------------------------ Consider the following variables:
+var lowMantra = mantra.toLowerCase();
+
+// // 1a. Write the code that determines if there is a 'B' in mantra.
+        
+        if (lowMantra.includes("b")){
+            console.log("true")
+        } else {
+            console.log("false")
+        }
+        
+        
+// // 1b. Write the code that determines if there is an 'x' in mantra.
+
+    if(lowMantra.includes("x")){
+        console.log("true")
+    } else {
+        console.log("false")
+    }
+
+// // 1c. Write the code that determines if there is a 'v' in mantra.
+
+    if(lowMantra.includes("v")){
+        console.log("true")
+    } else {
+        console.log("false")
+    }
+
```

Comment:
I like that you thought this through and added more code, but the includes method will return a boolean value on its own. This may be an opportunity to either: 1) add more info to your log or 2) remove it and shorten your code. Just a thought.